js/mail.js

```diff
@@ -920,5 +920,19 @@ $(document).ready(function () {
 	$(document).on('show', function() {
 		Mail.UI.changeFavicon(OC.filePath('mail', 'img', 'favicon.png'));
 	});
+	
+	// Listens to key strokes, and executes a function based on the key combinations.
+	$(document).keyup(function(event){
+	
+		var key = event.keyCode || event.which;
+		if (Mail.State.currentMessageId) {
+			switch (event.keyCode) {
+				case 46:
+					// Delete key
+					$('div.mail_message_summary.active div.icon-delete.action.delete').click();
```

Comment:
For a review notice like that only one reviewer needs to bring it up. ;) The two-reviewer policy is only for the thumbs-up since that’s the barrier for it to get merged.